src/components/Authorization/Authorization.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { login } from "../../service/auth/auth.service";
+
+import style from "./authorization.module.scss";
+import commonStyles from "../../common.module.scss";
+
+const Authorization = ({ isAuthorized, setIsAuthorized }) => {
+  const authorize = async () => {
+    try {
+      await login();
+      setIsAuthorized(true);
+    } catch (err) {
+      setIsAuthorized(false);
+    }
+  };
```

Comment:
redundant check, you could just create middleware that will handle 401 responses and redirect to login page